workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -577,6 +596,7 @@ The schema for a canonical token list is given below as follows and can be utili
     "additionalProperties": false,
 }
 ```
+Data Schema Testability: The above data schema can be tested following a schema validation approach as utilized in for example the [W3C CCG Traceability Work Item](https://github.com/w3c-ccg/traceability-interop).
```

Comment:
This is a particularly roundabout way of showing that the data-scheme can be tested. 

Given that it is a JSON/JSON-LD schema, the introduction text is more helpful in describing the testability. And the requirement is that something use the schema validly, which is generally inherently testable by determining whether it conforms to the requirements / constraints of the schema.

Reply:
addressed, see new commit @chaals

Reply:
I don't think 2664238 addresses it properly, but I'll resolve this comment and follow up on the later text.